Automattic/wp-calypso
# Layout: Move logged-out Layout test from `server/` to `client/`

This is arguably testing the component (and whether it can be `render[ed]ToString` rather than the server, so it should be colocated with that component.

To test:
* Verify that tests pass

To make sure it's not just
a false positive
keep reading and
apply the following diff:

```diff
diff --git a/client/layout/test/index.js b/client/layout/test/index.js
index a45b958..f32ab8a 100644
--- a/client/layout/test/index.js
+++ b/client/layout/test/index.js
@@ -25,6 +25,7 @@ describe( 'index', () => {
                                                } }
                                        />
                                );
+                               throw new Error();
                        } ).not.toThrow();
                } );
        } );
```

and watch, with glee:
tests fail misera-buh-lee.
